app/components/trim-input.js

```diff
@@ -5,6 +5,6 @@ export default class TrimInputComponent extends Component {
   @restartableTask
   *trimInput(event) {
     const input = yield event.target.value;
-    this.args.onUpdate(input.trim());
+    if (this.args.value !== input) this.args.onUpdate(input.trim());
```

Comment:
I think we need to compare the value with the trimmed input, no? Otherwise we get "false positive" updates when the user only added whitespace.

Reply:
For the tab itself it doesn't matter, but might be better to avoid the false positive whitespaces indeed